### DISCUSSION TITLE
Live Icon added to Paste

### DISCUSSION THREAD

**Discussion by corinne-nagel** [twilio-labs/paste#1921] [Request: Icon] [answered]
Hey hey,

Is the icon self-service not a thing anymore? ..."Self-service icon additions are temporarily unavailable"... The video team needs the Live product icon added to Paste so it can be used in the nav. Here's [the link to it in Bynder](https://library.twilio.com/media/?mediaId=9442A84C-6A25-4188-A352A91452123387
), svg is in "info" in additional files.

Unfortunately, we'll need this ASAP as the team needs it for Signal. Let me know if self-service is possible.

Thanks!

**Comment by SiTaggart** [comment]
Hi @corinne-nagel 

Self service is still possible, we just haven't gotten around to updating the docs after the Figma migration. I'm going to push them back up tomorrow.

For now, you can just fork the Figma library and add it there. The team will do the rest to pull it back in.

The eng team can then add the icon themselves, but honestly if it's for Signal it might be just as quick to send me the svg after you've exported it from Figma and I can get it in quicker.

**Reply by corinne-nagel** [reply]
Cool, here's the svg let me know if you run into any issues uploading and I can share with the team when you get it uploaded. Thanks!
[Paste Components.zip](https://github.com/twilio-labs/paste/files/7323858/Paste.Components.zip)

**Comment by SiTaggart** [comment]
PR up https://github.com/twilio-labs/paste/pull/1938

**Comment by SiTaggart** [comment, marked as answer]
Hi @corinne-nagel 

That icon is now published and ready to use in `@twilio-paste/icons@5.7.0`. https://paste.twilio.design/icons
